shopify/assets/logs/shopify_tests.yaml

```diff
@@ -0,0 +1,1531 @@
+id: shopify
+tests:
+ -
```

Comment:
I think the current samples do not provide the `service` field which is used to pick one of the nested pipelines defined, and as a result, none of the processors get applied.

Could you update the samples to cover the 4 different use-cases?
 - service:event
 - service:order
 - service:customer
 - service:product